### PR TITLE
Goal Detail Accordion updated and conflicts fixed

### PR DESCRIPTION
**Pull request by Ariiieee**
updated files and conflicts fixed

### CODE REVIEW

**Comment by abiona01** [comment]
@tundealabi review

File: package.json [hunk 1]
```diff
@@ -22,9 +22,13 @@
   "author": "Team Plato",
   "license": "MIT",
   "dependencies": {
+<<<<<<< HEAD
```
Comment (Adeniyii) [inline package.json:25]: here

File: package.json [hunk 2]
```diff
@@ -22,9 +22,13 @@
...
+    "@material-ui/icons": "^4.11.2",
+=======
```
Comment (Adeniyii) [inline package.json:27]: here

File: package.json [hunk 3]
```diff
@@ -22,9 +22,13 @@
...
     "@emotion/react": "^11.4.1",
     "@mui/icons-material": "^5.0.1",
     "@mui/material": "^5.0.2",
+>>>>>>> 1d17d85f3d15f3bef32b4d15a35c8ef48d935a0e
```
Comment (Adeniyii) [inline package.json:31]: here

File: client/yarn.lock [hunk 4]
```diff
@@ -2020,13 +2020,16 @@
     "@mui/utils" "^5.0.1"
     prop-types "^15.7.2"
 
+<<<<<<< HEAD
```
Comment (Adeniyii) [inline client/yarn.lock:2023]: here

File: yarn.lock [hunk 5]
```diff
@@ -228,13 +228,22 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+"@babel/runtime@^7.4.4":
+=======
 "@babel/runtime@^7.13.10", "@babel/runtime@^7.15.4", "@babel/runtime@^7.5.5", "@babel/runtime@^7.8.7":
+>>>>>>> 1d17d85f3d15f3bef32b4d15a35c8ef48d935a0e
```
Comment (Adeniyii) [inline yarn.lock:236]: here

**Review by Adeniyii — CHANGES_REQUESTED** [review]
Hi, there are merge highlights all over your code, please review, thanks.

**Pull request closed without merging**